maintainers/maintainer-list.nix

```diff
@@ -5138,6 +5138,12 @@
     githubId = 15957973;
     name = "Jeffry Molanus";
   };
+  gilice = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
thanks. Irrelevant now, because an other PR of mine was merged & added me as a maintainer